include/defines.h

```diff
@@ -12,15 +12,22 @@
 #define LONGITUDE_ONE_DEGREE_VALUE 10000000000.0
 #define ALTITUDE_UNAVAILABLE_VALUE (-2147483648)
 #define ALTITUDE_ONE_METER_VALUE 100.0
+#define ROTATION_ONE_DEGREE_VALUE 100.0
 #define DATE_UNAVAILABLE_VALUE 4294967295
 #define GPS_WEEK_UNAVAILABLE_VALUE 65535
 #define GPS_SECOND_OF_WEEK_UNAVAILABLE_VALUE 4294967295
 #define MAX_WAY_DEVIATION_UNAVAILABLE_VALUE 65535
 #define MAX_WAY_DEVIATION_ONE_METER_VALUE 1000.0
 #define MAX_LATERAL_DEVIATION_UNAVAILABLE_VALUE 65535
 #define MAX_LATERAL_DEVIATION_ONE_METER_VALUE 1000.0
-#define MIN_POSITIONING_ACCURACY_NOT_REQUIRED_VALUE 0
-#define MIN_POSITIONING_ACCURACY_ONE_METER_VALUE 1000.0	// ISO specification unclear on this value
+#define MAX_YAW_DEVIATION_UNAVAILABLE_VALUE 65535
+#define MAX_YAW_DEVIATION_ONE_DEGREE_VALUE 100.0
+#define MIN_POSITIONING_ACCURACY_NOT_REQUIRED_VALUE 65535
+#define MIN_POSITIONING_ACCURACY_ONE_METER_VALUE 100.0
+#define COMMUNICATION_TIMEOUT_ONE_SECOND_VALUE 100.0
+#define MONR_RATE_ONE_HZ_VALUE 1.0
```

Comment:
Decimal Hertz?
Monr, Monr2, Heab is inreased by 1

Reply:
> Monr, Monr2, Heab is inreased by 1
What does this mean?

Yes, Hz is decimal since these values are used for converting from millimetres, centiseconds, centidegrees etc. into doubles with unprefixed units. These values are 1 since the ISO unit is simply Hz for this value (but I put them there anyway so the library follows the same structure).